docs/network protocol.md

```diff
@@ -554,6 +554,7 @@ class JSONMessagePart(TypedDict):
     color: Optional[str] # only available if type is a color
     flags: Optional[int] # only available if type is an item_id or item_name
     player: Optional[int] # only available if type is either item or location
+    hint_status: Optional[int] # only available if type is hint_status
```

Comment:
I guess this should be Optional[HintStatus] rather than int

Reply:
```suggestion
    hint_status: Optional[[HintStatus](#HintStatus)] # only available if type is hint_status
```
aye, that would seem more correct.